install.sh

```diff
@@ -362,7 +292,7 @@ parse_flags ()
                 with_qt='1'
                 ;;
             --without-qt)
-                with_qt='0'
+                with_qt=''
```

Comment:
This change is wrong. It makes always ask user for installing Qt GUI deps even when specified not to do it. Possible values of `with_qt` is: '1' - install GUI, '0' - don't install GUI, '' - ask (default).

Reply:
Ok. Will restore this.